File: corelib/src/main/java/org/apache/cordova/repository/DatasetRepository.java
package org.apache.cordova.repository;

import android.content.Context;

import org.apache.cordova.domen.Dataset;

public interface DatasetRepository {

    interface RepoCallback {
        void successResponse(Dataset value);

        void handleError(String message);
    }


    void getConfig(Context context); //String rawUrl
}

File: kwk/StelthCore/src/main/java/org/apache/cordova/repository/DatasetRepository.kt
package org.apache.cordova.repository

import android.content.Context
import org.apache.cordova.domen.UIVisibleDataset

interface DatasetRepository {
    interface RepoCallback {
        fun successResponse(value: UIVisibleDataset)

        fun handleError(message: String?)
    }


    fun getConfig(context: Context?) //String rawUrl
}
